_layouts/default.html

```diff
@@ -92,7 +92,7 @@
                                 <p><a href="http://lists.webkit.org/mailman/listinfo/webkit-help">webkit-help</a></p>
                                 <p><a href="http://lists.webkit.org/mailman/listinfo/webkit-dev">webkit-dev</a></p>
                             </li>
-                            <li><span class="subheader">&#8227; #webkitgtk on freenode</span></li>
+                            <li><span class="subheader">&#8227; #webkitgtk on irc.gnome.org</span></li>
```

Comment:
Shouldn't this also mention the [#webkitgtk:matrix.org](https://matrix.to/#/%23webkitgtk:matrix.org) room?

Reply:
Sure

```suggestion
                            <li><span class="subheader">&#8227; <a href="https://matrix.to/#/%23webkitgtk:matrix.org">#webkitgtk:matrix.org</a></span></li>
                            <li><span class="subheader">&#8227; #webkitgtk on irc.gnome.org</span></li>
```

Should probably test this before merging though.

Reply:
Tested, works.

Reply:
I guess we can make a follow-up PR with that, let's merge this as-is.